src/Core/Tool/Validator.php

```diff
@@ -11,67 +11,21 @@
 
 namespace Ushahidi\Core\Tool;
 
-use Ushahidi\Core\Entity;
-use Ushahidi\Core\Traits\GetSet;
-use Ushahidi\Core\Tool\Validation;
-use Ushahidi\Core\Tool\ValidationEngineTrait;
-
-abstract class Validator
+interface Validator
```

Comment:
This is the key change in this PR. A validator really only needs to implement 2 functions, the rest was all Kohana related